messages/de/SpaceModule.base.php

```diff
@@ -106,7 +106,7 @@
   'Shown on About Page.' => 'Wird auf Informationsseite angezeigt.',
   'Sorry, you are not allowed to leave this space!' => 'Sorry, aber du kannst diesen Space nicht verlassen.',
   'Sorting' => 'Sortierung',
-  'Space' => 'Space',
+  'Space' => 'Gruppen und Vereine',
   'Space Visibility' => 'Space Sichtbarkeit',
   'Space directory' => 'Space-Verzeichnis',
   'Space is invisible!' => 'Space ist unsichtbar!',
```

Comment:
what about other "Space" occurrences?

Reply:
what about it?

Reply:
should this become something like "Gruppen und Vereine"?

Reply:
waiting of Mihaela feedback I guess ..

Reply:
ok, we will need to change other at a later point